lib/ClimaCoreTempestRemap/src/wrappers.jl

```diff
@@ -98,7 +98,11 @@ function remap_weights(
     --out_map $weightfile
     ```
     for (k, v) in kwargs
-        append!(cmd.exec, [string("--", k), string(v)])
+        if isempty(v)
```

Comment:
what would `v` be in this case?

Reply:
`v` would be the value you want to set argument `k` to in the command we're constructing. For example, `k=in_np` and `v=1` would produce the string `--in_np 1`, which would get appended to `cmd`.

Reply:
I meant what would the user set it to if they wanted to use the mono flag?